tests/compute/test_dis_sampler.py

```diff
@@ -0,0 +1,49 @@
+from dgl import backend as F
+import numpy as np
+import scipy as sp
+import dgl
+from dgl import utils
+
+import os
+import time
+
+def generate_rand_graph(n):
+    arr = (sp.sparse.random(n, n, density=0.1, format='coo') != 0).astype(np.int64)
+    return dgl.DGLGraph(arr, readonly=True)
+
+def start_trainer():
+    g = generate_rand_graph(100)
+    recv = dgl.contrib.sampling.SamplerReceiver(ip="127.0.0.1", port=50051)
+    nodeflow = recv.Receive(g)
+    print("----- trainer ------")
+    print(nodeflow._node_mapping.todgltensor())
+    print(nodeflow._edge_mapping.todgltensor())
+    print(nodeflow._layer_offsets)
+    print(nodeflow._block_offsets)
+    print("----- trainer ------")
```

Comment:
I think it's better to add checks in the unit test instead of just printing them.